src/transmuxer-worker.js

```diff
@@ -119,6 +119,10 @@ class MessageHandlers {
     this.transmuxer.setBaseMediaDecodeTime(Math.round(timestampOffset * 90000));
   }
 
+  setAudioAppendStart(data) {
+    this.transmuxer.setAudioAppendStart(Math.round(data.appendStart * 90000));
```

Comment:
Should this be `Math.ceil`?? Since we take so much care elsewhere to make sure we don't overlap existing data, the possibility of this one rounding down the buffer seems a little worrying.